sql/moz-fx-data-shared-prod/internet_outages/global_outages_v1/schema.yaml

```diff
@@ -0,0 +1,49 @@
+fields:
```

Comment:
For testing this I ended up creating the schemas for all downstream dependencies of clients_daily